hkmc2/shared/src/test/mlscript/codegen/ReboundLet.mls

```diff
@@ -15,7 +15,7 @@ foo()
 
 let x = 1
 let f = () => x
-//│ f = [function f]
+//│ f = [function lambda]
```

Comment:
Hm. Did this actually rewrite it as a proper function before? Or is this just due to the way it's printed?

Reply:
Any suggestions on how this should be changed? To be honest I think it makes more sense here to print `[function anonymous]` or something, but I don't know

Reply:
In JS, you can remove the name in 2 ways:
```js
let x = (0, () => {}); // This makes the lambda to occur inside an expression, which disable the automatic name detection)
let y = () => {};
// set the "name" field to empty. Note that y.name = ""; does not work, as the field is not writable
// but we can still write to not writable field using the following:
Object.defineProperty(y, "name", { value: "", writable: false, enumerable: false, configurable: false});
```

Reply:
btw, both of the ways work exactly the same for proper anon functions (`function () {}`) and anon arrow functions (`() => {}`)

Reply:
Seems like a much better idea to use the first approach. Messing with writable fields in lamdas is exactly the kind of things that destroy JIT performance (as it's likely to make it throw its hands in the air and say "A lambda with overridden fields? All right this is some JS nonsense, let's not even try to optimize it...")